src/flowcean/core/report.py

```diff
@@ -135,6 +137,68 @@ def pretty_print(
             )
             console.print(panel)
 
+    def select_best_model(
+        self,
+        metric_name: str,
+        *,
+        mode: str = "maximize",
+    ) -> Model:
+        """Return the best model according to a metric.
+
+        Args:
+            metric_name: Name of the metric to select by.
+            mode: One of {"maximize", "minimize", "max", "min"}.
+
+        Returns:
+            The best Model instance. Requires that a mapping of models was
+            attached to the report under `models_by_name` (as done by
+            evaluate_offline).
```

Comment:
This is exactly the inverse requirement. The report should either carry this information always, or the function requires this mapping as an argument